avm/ptn/sa/conversation-knowledge-mining/main.bicep

```diff
@@ -0,0 +1,1168 @@
+      ]
+    }
+    azCliVersion: '2.52.0'
+    primaryScriptUri: '${varCKMGithubBaseUrl}infra/scripts/run_create_index_scripts.sh'
+    arguments: '${varCKMGithubBaseUrl} ${avmKeyVault.outputs.name}'
+    timeout: 'PT1H'
+    retentionInterval: 'PT1H'
+    cleanupPreference: 'OnSuccess'
+  }
+  dependsOn: [moduleAIFoundry, avmSQLServer, avmDeploymentScriptCopyData]
+}
+
+//========== Azure function: Managed Environment ========== //
+
+module avmFunctionsManagedEnvironment 'br/public:avm/res/app/managed-environment:0.10.0' = {
+  name: format(avmDeploymentNameFormat, varFunctionsManagedEnvironmentResourceName)
+  params: {
+    name: varFunctionsManagedEnvironmentResourceName
+    tags: tags
+    location: functionsManagedEnvironmentLocation
+    enableTelemetry: enableTelemetry
+    zoneRedundant: false
+    workloadProfiles: [
+      {
+        workloadProfileType: 'Consumption'
+        name: 'Consumption'
+      }
+    ]
+    publicNetworkAccess: 'Enabled'
+    peerTrafficEncryption: false
+    logAnalyticsWorkspaceResourceId: avmLogAnalyticsWorkspace.outputs.resourceId
+  }
+}
+
+// NOTE: This version of the AVM module does not support the deployment to Azure Container Apps Environment resource
+// ERROR: VnetRouteAllEnabled cannot be configured for function app deployed on Azure Container Apps. Please try to configure from Azure Container Apps Environment resource
+
+// module avmFunctionCharts 'br/public:avm/res/web/site:0.15.0' = {
+//   name: format(avmDeploymentNameFormat, varFunctionChartsResourceName)
+//   params: {
+//     name: varFunctionChartsResourceName
+//     tags: tags
+//     location: functionChartsLocation
+//     enableTelemetry: enableTelemetry
+//     diagnosticSettings: [{ workspaceResourceId: avmLogAnalyticsWorkspace.outputs.resourceId }]
+//     kind: 'functionapp,linux,container,azurecontainerapps'
+//     managedIdentities: {
+//       systemAssigned: true
+//     }
+//     managedEnvironmentId: avmFunctionsManagedEnvironment.outputs.resourceId
+//     serverFarmResourceId: 'null'
+//     siteConfig: {
+//       linuxFxVersion: 'DOCKER|${functionChartDockerImageContainerRegistryUrl}/${functionChartDockerImageName}:${functionChartDockerImageTag}'
+//       functionAppScaleLimit: functionChartAppScaleLimit
+//       minimumElasticInstanceCount: 0
+//     }
+//     appSettingsKeyValuePairs: {
+//       AzureWebJobsStorage_accountname: moduleAIFoundry.outputs.storageAccountName
+//       SQLDB_DATABASE: varSqlServerDatabaseName
+//       SQLDB_PASSWORD: sqlServerAdministratorPassword
+//       SQLDB_SERVER: avmSQLServer.outputs.fullyQualifiedDomainName
+//       SQLDB_USERNAME: sqlServerAdministratorLogin
+//     }
+//     // workloadProfileName: 'Consumption'
+//     // resourceConfig: {
+//     //   cpu: functionChartCpu
+//     //   memory: functionChartMemory
+//     // }
+//     storageAccountRequired: false
+//     appInsightResourceId: moduleAIFoundry.outputs.applicationInsightsResourceId
+//   }
+// }
```

Comment:
Can this be removed

Reply:
I guess it's here for the moment when the AVM module can support it, and the code below is a workaround for the time being